modules/network-graph-20-view.adoc

```diff
@@ -13,22 +13,47 @@ The network graph provides high-level and detailed information about deployments
 
 Network components:: From the top menu, you can select namespaces (indicated by the *NS* label) and deployments (indicated by the *D* label) to display on the graph for a chosen cluster (indicated by the *CL* label). You can further filter deployments by using the drop-down list and selecting criteria on which to filter, such as common vulnerabilities and exposures (CVEs), labels, and images.
 External entities:: These represent entities that are connected outside of your cluster. For more information, see "External entities and connections in the network graph".
+Internal entities:: These represent entities that are connected inside of your cluster and are not considered external. Internal entities are displayed when Sensor is unable to match connections to existing deployments, but the involved IP address is part of the cluster's internal networks. Based on the IP addresses, the system has determined that those flows do not leave the cluster. For more information about the kind of connections that are considered internal, see "Connections involving internal entities".
 Network policies:: You can view existing policies for a selected component or view components that have no policies.
 Network flows:: You can select one of the following flows for the graph:
 Active traffic:: Selecting this default option shows observed traffic, focused on the namespace or specific deployment that you selected. You can select the time period for which to display information.
 Inactive flows:: Selecting this option shows potential flows allowed by your network policies, helping you identify missing network policies needed to achieve tighter isolation. You can select the time period for which to display information.
 
 You can also simulate network policies from the network graph view. See "Simulating network policies from the network graph" for more information.
 
-[discrete]
-== Navigation and user interface in the network graph
+[id="navigation-user-interface_{context}"]
+== Displays, navigation, and the user interface in the network graph
+
+The network graph, shown in the following graphic, provides the ability to click on items and view additional information about them. You can also perform actions within the graph such as adding a network flow to your baseline.
```

Comment:
Minor suggestion to be a little more focused on the reader

```suggestion
The network graph, shown in the following graphic, allows you to click on items and view additional information about them. You can also perform actions within the graph, such as adding a network flow to your baseline.
```

Or you could do it even more:

```suggestion
You can use the network graph, shown in the following graphic, to click on items and view additional information about them. You can also perform actions within the graph, such as adding a network flow to your baseline.
```